committers.md

```diff
@@ -53,6 +53,7 @@ navigation:
 |Cheng Lian|Databricks|
 |Yanbo Liang|Facebook|
 |Jungtaek Lim|Cloudera|
+|Gabor Somogyi|Cloudera|
```

Comment:
Actually there's a convention in the list if I understand correctly (I don't know where this is documented) - the list is alphabetically sorted by "last name" (not first name).

Could you please check the sequence a bit and please update the same if the list follows the pattern?

Reply:
Oh, not seen pattern in the first letter so thought the order doesn't matter, checking...